0.9 JS Fundamentals - DOM/Add blog.js

```diff
@@ -1,4 +1,6 @@
-document.addEventListener('DOMContentLoaded', async () => {
+document.addEventListener('DOMContentLoaded', fetchData());
```

Comment:
do we need to call the function here? I mean fetchData()? I thing we have to give only ref. and it will call once event triggered

Reply:
Yes we need to add callback function to invoke particular event